deploy/playbooks/roles/common/tasks/ssl.yml

```diff
@@ -0,0 +1,99 @@
+---
+
+- name: ensure ssl certs directory
+  action: file dest=/etc/ssl/certs state=directory
+  sudo: true
+
+- name: ensure ssl private directory
+  action: file dest=/etc/ssl/private state=directory
```

Comment:
I thought I had fixed that one 

![5dr730dy0u8x](https://cloud.githubusercontent.com/assets/317847/16965130/b9fcf68c-4dcc-11e6-9157-a6c32a9b5eba.gif)